src/vm/jitinterface.cpp

```diff
@@ -11443,7 +11443,21 @@ void CEEJitInfo::recordRelocation(void * location,
                 PCODE jumpStubAddr = ExecutionManager::jumpStub(m_pMethodBeingCompiled,
                                                                 (PCODE)  target,
                                                                 (BYTE *) loAddr,
-                                                                (BYTE *) hiAddr);
+                                                                (BYTE *) hiAddr,
+                                                                NULL,
+                                                                false);
+
+                // Keep track of conservative estimate of how much memory may be needed by jump stubs. We will use it to reserve extra memory
+                // on retry to increase chances that the retry succeeds.
+                m_reserveForJumpStubs = max(0x400, m_reserveForJumpStubs + 2*BACK_TO_BACK_JUMP_ALLOCATE_SIZE);
```

Comment:
You said conservative so i used 2x.  I figured ask for twice as much as failed...

Looks like you used `BACK_TO_BACK_JUMP_ALLOCATE_SIZE` + 4 above.  Hard coded as `0x10`.  

I also used your hardcoded `0x400` which I suspect should be something else....